targets/unity-v2/Testing/Tests/Entity/EntityApiTests.cs

```diff
@@ -14,8 +16,10 @@ public class EntityApiTests : UUnitTestCase
         private const string TEST_OBJ_NAME = "testCounter";
         // Test variables
         private EntityKey _entityKey;
-        private string _testFileUrl;
-        private string _testFileChecksum;
+        private string _testFileData;
+        private const string TEST_FILE_NAME = "testfile";
+        private const string _testPayload = "{123456789}";
+        private UUnitTestContext _testContext;
```

Comment:
This is really unsafe.
This is a great way for the testContext of one test to leak into another.
The testContext instance is passed into the constructor of each test specifically to avoid any potential leakage like this.
If you only have one test in your test class, it would be... LESS dangerous, but this isn't the case even here.

Avoiding that leakage is exactly why I created wrappers like ApiActionWrapper.

Reply:
I'm not sure how exactly leak can happen, I removed it anyway.